package.json

```diff
@@ -21,8 +21,9 @@
     "test": "jest"
   },
   "dependencies": {
-    "eosjs": "20.0.0",
-    "eosjs-ecc": "4.0.4",
+    "@types/elliptic": "^6.4.10",
+    "elliptic": "^6.5.0",
```

Comment:
These should both be version-locked; they are not "devDependencies"

Reply:
Bump, this still should be addressed in this PR.